README.md

```diff
@@ -47,3 +47,25 @@ actions on push.
 Specific hooks can be [temporarily disabled](https://pre-commit.com/#temporarily-disabling-hooks).
 
 You can install hooks only for a specific step (i.e. [pre-push](https://pre-commit.com/#pre-commit-during-push)).
+
+# Testing in Snoodev
```

Comment:
🔕 Not against it here, and these instructions seem fine, but seems a little strange to me for this to be here but testing sidecars to be [in the wiki](https://reddit.atlassian.net/wiki/spaces/BAS/pages/2410939140/Baseplate+Sidecars#How-do-I-test-the-sidecars%3F)

Reply:
Note that talks about how to release sidecars and then test them, not how to test them locally. Also currently this approach doesn't work with sidecars.

Reply:
🔕 nit: Multiple top-level headings in the same document